privaterelay/templates/profile.html

```diff
@@ -16,8 +16,9 @@
 {% get_social_accounts request.user as accounts %}
 
 <main id="profile-main" class=" bg-light dashboard-container " data-api-token="{{ user_profile.api_token }}">
-  {% include "includes/messages.html" %}
-
+  <div class="messages-profile">
+    {% include "includes/messages.html" %}
```

Comment:
Placed the `messages` partial into a `div` for the white background on the profile page.